mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -657,6 +673,13 @@ class DistributedLDAModel private[clustering] (
     }
   }
 
+  /** Java-friendly version of [[topTopicsPerDocument]] */
+  def javaTopTopicsPerDocument(
```

Comment:
Oh OK, try running this from the command line:
```
javap -classpath assembly/target/scala-2.10/spark-assembly-1.5.0-SNAPSHOT-hadoop2.2.0.jar org.apache.spark.mllib.clustering.DistributedLDAModel
```
(Replace the jar filepath with the location on your computer.)

That should print the class signature, including all methods.